services/brig/deb/opt/brig/templates/team/en/email/invitation.txt

```diff
@@ -14,5 +14,3 @@ Please don't reply to this message.
 
 
 (c) Wire Swiss GmbH
-
-Privacy Policy  |  Report misuse
```

Comment:
Not sure if we have to remove them.. all add the actual links

```
Privacy Policy - https://wire.com/...
Report misuse - https://....
```

Reply:
Discussed with Yorgos, they're not present in several other plain text messages, so it seemed best to remove them entirely in this context.

The linked variants will remain in the HTML versions.